### DISCUSSION TITLE
Folk the other repository

### DISCUSSION THREAD

**Discussion by crypto-demigod** [community/community#33401] [Apps, API and Webhooks] [answered]
How can I folk the other's repositories?

**Comment by massimo1220** [comment, marked as answer]
hi, @superherodemigod.

I read your question. it's simple.

You can access the other's public repositories then look right top. there is a "folk" button.
you can just push this button to folk the other's repositories.

**Reply by crypto-demigod** [reply]
thanks.
but is it okay to folk the others?

**Reply by massimo1220** [reply]
good question.
it's okay to folk the other's public repositories but you can't folk private repositories.

**Reply by crypto-demigod** [reply]
I see. thanks.

**Reply by massimo1220** [reply]
my pleasure.
